gradle.properties

```diff
@@ -5,9 +5,9 @@ android.useAndroidX=true
 #enabledSandboxResourceProcessor=false
 enabledSandboxResourceProcessor=true
 enableKotlinNative=true
-enableKotlinNativeLinux=true
```

Comment:
to revert later

Reply:
Did you have any problems with this too?

Reply:
Not sure why i did this, may be to avoid IntelliJ to create/compile linux native module as i working only on osx/ios target.